output/opendap_post2.sh

```diff
@@ -811,9 +818,9 @@ SSHCMD
       threddsPostStatus=fail
       MSGS="$SCENARIO: $_THIS: The opendap post method $OPENDAPPOSTMETHOD was not recognized."
       if [ "$MANUAL" == 1 ]; then
-        echo "$MSG"
+         echo "$MSG"
```

Comment:
seems like a leading white space was added, other lines have this above also

Reply:
This script has a couple different indentation schemes going on, I will revert and commit/push.